### PR TITLE
Create crvt.json

### PR DESCRIPTION
**Pull request by slovakians**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://slovakians.github.io/crvts/

### CODE REVIEW

File: domains/crvt.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "portfolio site",
+  "owner": {
+    "username": "slovakians",
+    "email": "siresirol937@gmail.com"
+  },
+  "record": {
+    "CNAME": ["slovakians.github.io/crvt/"]
```
Comment (DEV-DIBSTER) [inline domains/crvt.json:8]: CNAME shouldn't be in a array. CNAME should be pure hostname.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/crvt.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "portfolio / profile page",
+    "repo": "https://github.com/slovakians/crvts",
+    "owner": {
+        "username": "Slovakians",
+        "email": "siresirol937@gmail.com",
+        "twitter": "<twitter-username>"
```
Comment (orxngc) [inline domains/crvt.json:7]:
```suggestion
```
Either put your Twitter username here or delete the line entirely.

Comment (slovakians) [reply domains/crvt.json:7]: oh opps wrong link

File: domains/crvt.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "portfolio / profile page",
+    "repo": "https://github.com/slovakians/crvts",
+    "owner": {
+        "username": "Slovakians",
+        "email": "siresirol937@gmail.com",
```
Comment (CuteDog5695) [inline domains/crvt.json:6]:
```suggestion
        "email": "siresirol937@gmail.com"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Seems fine to me now

**Pull request merged**